bcipy/acquisition/devices.json

```diff
@@ -46,6 +46,14 @@
         "connection_methods": [ "LSL" ],
         "description": "Alias to GTec g.USBamp for backwards compatibility and demos."
     },
+    {
+        "name": "Tobii Nano",
+        "content_type": "Eyetracker",
```

Comment:
Hopefully this will soon be changed to 'Gaze'. https://github.com/labstreaminglayer/App-TobiiPro/issues/6

Reply:
Do you think I should merge it with type Gaze?

Reply:
Yes, go ahead and merge. When we updated the executable on the laptop we can change it then.